Test/verified_test.py

```diff
@@ -398,4 +398,11 @@
 >>> cm2 = ConfusionMatrix(matrix = {1:{1:2,0:3},0:{0:775,1:4}})  # Verified Case -- (https://bit.ly/3BVdNBp)
 >>> cm2.BB[1]
 0.3333333333333333
+>>> cv = Curve(actual_vector = np.array([1, 1, 2, 2]), probs = np.array([[0.1, 0.9], [0.4, 0.6], [0.35, 0.65], [0.8, 0.2]]), classes=[2,1])  # Verified Case -- (https://bit.ly/3MIMk9z)
```

Comment:
`actual_vector=np.array([1, 1, 2, 2]), probs=np.array([[0.1, 0.9], [0.4, 0.6], [0.35, 0.65], [0.8, 0.2]]), classes=[2, 1]`

Reply:
fixed in https://github.com/sepandhaghighi/pycm/pull/457/commits/b49a1624972ceb4b627263ee299fb25350e8319d